authors/verah-ombui/index.md

```diff
@@ -0,0 +1,5 @@
+---
+title: Verah Ombui
+type: engineering-education/author
+---
+Verah Ombui is a undergraduate Computer Science student at Jomo Kenyatta University Of Agriculture and Technology. His interests are web development, cloud computing, and data analysis science. Verah is  a Technology enthusiast.
```

Comment:
Verah Ombui is `an` ...  also is Verah a girl `His interests are web development,`